src/nurse/tests/test_views.py

```diff
@@ -784,3 +784,55 @@ def test_post_only_staff(self, user, client):
         assert response.json() == {
             "detail": "You do not have permission to perform this action."
         }
+
+
+@pytest.fixture
+def user_profile(user):
+    return UserOneSignalProfile.objects.create(user=user, subscription_id="123456789")
+
+
+@pytest.fixture
+def another_user_profile(another_user):
+    return UserOneSignalProfile.objects.create(
+        user=another_user, subscription_id="987654321"
+    )
+
+
+@pytest.fixture
+def another_user(db):
+    return User.objects.create_user(
+        username="username2", email="user2@example.com", password="testpass123"
+    )
```

Comment:
Ideally use yield, refs:
https://docs.pytest.org/en/8.0.x/how-to/fixtures.html#yield-fixtures-recommended

Reply:
I have delete return is added yield